test/org/loklak/harvester/TwitterScraperTest.java

```diff
@@ -33,7 +31,7 @@ public void testPrepareSearchURL() {
             url = TwitterScraper.prepareSearchURL(query[i]);
 
             //compare urls with urls created
-            assertEquals(url, out_url[i]);
+            assertThat(out_url[i], is(url));
```

Comment:
But we must keep up with the conventions. The expected output should go in as the second argument to `assertThat`.